Sources/KlaviyoSwift/Vendor/AnyEncodable.swift

```diff
@@ -0,0 +1,303 @@
+/**
```

Comment:
will we have to keep this updated in any way?

Reply:
Probably not - it's mostly base data types. I can make a regular dependency but it had other stuff I didn't need.

Reply:
We need to encode [String: Any] stuff.